pkg/recipes/terraform/config/providers/aws_test.go

```diff
@@ -45,24 +50,28 @@ type mockAWSCredentialsProvider struct {
 func newMockAWSCredentialsProvider() *mockAWSCredentialsProvider {
```

Comment:
should we write unit tests for irsa here as well?

Reply:
yes, I am going to add UT to aws_test as part of PR that supports terraform.

Reply:
same as https://github.com/radius-project/radius/pull/7739#discussion_r1679825544